aidants_connect_habilitation/templates/_recap_general_info.html

```diff
@@ -0,0 +1,72 @@
+<h2>Information générales</h2>
+<div class="fr-grid-row fr-grid-row--gutters fr-mb-4v">
+  <div class="fr-col-12 fr-col-md-4">
+    <h3 class="fr-h5">Personne effectuant la demande</h3>
+  </div>
+  <div class="fr-col-12 fr-col-md-8">
+    <h3 class="fr-h5">Structure</h3>
+  </div>
+  <div class="fr-col-12 fr-col-md-4">
+    <div class="fr-bordered fr-p-4w flex flex-between flex-column fr-height-100 ">
+      <div>
+        <p>
+          <strong>{{ issuer.get_full_name }}</strong><br/>
+          <span class="fr-text-mention--grey fr-text--sm">{{ issuer.profession }}</span>
+        </p>
+        <p class="fr-text--sm">
+          {{ issuer.email }}<br>{{ issuer.phone }}
+        </p>
+      </div>
+      {% if show_edit_buttons %}
+        <div class="fr-mt-3w flex flex-right fr-link">
+          <a class="no-underline"
+            href="{% url 'habilitation_modify_issuer' issuer_id=issuer.issuer_id %}"
+          >
+            <span class="fr-icon-pencil-fill"></span>
+          </a>
+        </div>
```

Comment:
```suggestion
          <ul class="fr-btns-group fr-btns-group--sm fr-btns-group--right fr-btns-group--inline">
            <li>
            <a
              class="fr-btn fr-btn--tertiary fr-btn--icon fr-icon-pencil-fill"
              href="{% url 'habilitation_modify_issuer' issuer_id=issuer.issuer_id %}"
            >
              Éditer
            </a>
            </li>
          </ul>
```

Voir [les groupes de bouton du DSFR](https://www.systeme-de-design.gouv.fr/composants-et-modeles/groupe-de-boutons).